mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -1182,15 +1217,50 @@ async fn broadcast_my_swap_status(ctx: &MmArc, uuid: Uuid) -> Result<(), String>
     };
     status.hide_secrets();
 
+    let status = SwapStatus {
+        method: "swapstatus".into(),
```

Comment:
`method` can be a static str if it's intended to be filled manually like this.

Reply:
ummm, this is actually not possible because when we parse this msg from the network, we can't some how put it inside static str (such str was provided at runtime).
this line: https://github.com/KomodoPlatform/komodo-defi-framework/blob/c1bd112206d6ff275379bc2957686c566478c1ca/mm2src/mm2_main/src/lp_swap.rs#L366